ebuild-maintenance/removal/text.xml

```diff
@@ -83,6 +83,48 @@ Date:   Tue Oct 3 21:43:03 2017 +1100
   Closes: https://bugs.gentoo.org/629144
 </pre>
 
+</body>
+</section>
+
+<section>
+<title>Removing a virtual package</title>
+<body>
+
+<p>
+Virtual packages are generally removed when they have no more than one
+provider left. The removal is preceded by updating the remaining ebuilds
+not to use the virtual. Since virtuals do not install any files, there
+is little value in proactively forcing them to be uninstalled from user
+systems and therefore an alternative removal process is recommended.
+</p>
+
+<p>
+In order to remove a virtual package, follow the following procedure:
+</p>
+
+<ol>
+  <li>
+    Update all the remaining ebuilds not to reference the virtual.
+    Since there is no urgent need to remove the virtual from user
+    systems and the resulting rebuilds would be unnecessary, do not bump
+    ebuilds when replacing the dependency.
+  </li>
+
+  <li>
+    Remove the package directly.
+  </li>
+
+  <li>
+    Perform the post-removal cleanup, the same as with regular packages.
+  </li>
+</ol>
```

Comment:
By the style guide:
- No full stops if the item consists only of a single sentence
- No blank lines between items